packages/openapi-generator/src/sdk-metadata/generation-and-usage.ts

```diff
@@ -0,0 +1,61 @@
+import { first } from '@sap-cloud-sdk/util';
+import {
+  GenerationAndUsage,
+  InstructionWithText,
+  getLinks,
+  getSdkVersion,
+  apiSpecificUsageText,
+  genericUsageText,
+  getGenerationSteps
+} from '@sap-cloud-sdk/generator-common';
+import { OpenApiDocument } from '../openapi-types';
+import { apiSpecificCodeSample, genericCodeSample } from './code-sample';
+
+export async function getGenerationAndUsage(
+  openApiDocument: OpenApiDocument
+): Promise<GenerationAndUsage> {
+  return {
+    genericUsage: getGenericUsage(),
+    apiSpecificUsage: getApiSpecificUsage(openApiDocument),
+    links: getLinks(),
+    generationSteps: getGenerationSteps(
+      installCommand,
```

Comment:
if this is not used anywhere else just write it inline instead of a const?

Reply:
same would aplly to the OData version, but it is also a matter of taste if you use a constant or put it to inline.